### ISSUE TITLE
[Global] Input borders feature insufficient color contrast

### ISSUE THREAD

**Issue by svinkle** [CovidShield/portal#11]
## Issue summary

The color contrast for the form `input` borders is:

- Foreground: `#cfcfcf`
- Background: `#ffffff`
- Contrast ratio: `1.55:1`

<img width="735" alt="Screen Shot 2020-05-18 at 1 08 56 PM" src="https://user-images.githubusercontent.com/1392632/82240657-cb6e1a00-9908-11ea-9234-d2c235b39a82.png">

#### CSS

```css
.text-field__input,
.select {
  /* … */
  border: 1px solid #cfcfcf;
  /* … */
}
```

## Steps to reproduce

1. Go to https://portal-rails.myshopify.io/users/1/edit with with any supported browser
2. Login with `admin`/`password`
3. [Test the color contrast of the `input` borders](https://contrast-ratio.com/#cfcfcf-on-#ffffff)

### Expected

- Color contrast to meet AA requirements.

### Actual

- Color contrast does not meet AA requirements.

## Recommendation

Use a darker `border` color to increase the contrast ratio to at least `3.0:1`.

### Recommended code

```css
.text-field__input,
.select {
  /* … */
  border: 1px solid #949494;
  /* … */
}
```

## Specifications

- **Component:** Global
- **WCAG Principle:** Perceivable
- **WCAG SC:** 1.4.11 Non-text Contrast
- **Severity:** Medium
- **Effort:** Low
